code/stable/ssp/SRS/SSP_SRS.tex

```diff
@@ -237,6 +239,8 @@ \subsection{Table of Symbols}
 \\
 ${σ_{N}}'$ & Effective Normal Stress: The normal stress in a soil mass that is effective in causing volume changes; represents the average normal stress carried by the soil skeleton. & ${\text{Pa}}$
 \\
+$τ$ & Tangential Stress: The total force per area acting on the soil mass. & ${\text{Pa}}$
```

Comment:
Please change this definition to "The shear force per unit area."  We don't have the concept of total force for shear.  We also don't need to mention the soil mass.  This definition is true whether we are talking about soil, or any other material.

Reply:
@smiths If we are making an effort to keep these generic (i.e. by not mentioning soil mass, etc.), it may be worthwhile to move them (normal/tangential force/stress) to data-drasil instead of defining them directly in SSP, so they can be re-used by potential future examples. Thoughts?

Reply:
Yes, @bmaclach, moving the generic concepts to data-drasil makes sense.  I don't think we need to make this a high priority at the moment, since ssp is the only example with the concept of stress, but this is the way things should go in the future.  I'd like to see us attack this problem for all of the examples, using two different approaches:

1. Top-down - we organize the information using an ontology for physics, math and computing knowledge.  Ideally we would be able to find an existing ontology, instead of having to invent something.  Unfortunately, my preliminary efforts to find existing ontologies haven't been that encouraging.
2. Example driven - we add more examples and let the ontology build up organically.  For instance, the concept of stress applies to all continuum mechanics problems, but the concept of effective stress is reserved for continuum mechanics problems where the material is a soil, or other granular material.
I'll edit the "potential projects" wiki page to mention refactoring the existing Drasil examples as the ontology evolves.